scripts/sv_main.lua

```diff
@@ -14,7 +14,7 @@ end
 local apiHost = GetConvar("txAdmin-apiHost", "invalid")
 local apiToken = GetConvar("txAdmin-apiToken", "invalid")
 local txAdminClientVersion = GetResourceMetadata(GetCurrentResourceName(), 'version')
-if GetConvar('txAdminServerMode', 'false') ~= 'true' then
+if GetConvar('txAdmin-serverMode', 'false') ~= 'true' then
```

Comment:
This being renamed seems bad, game code checks for this in at least one place to enable certain Node.js behavior fixing issues with JS resources on the server abusing stdout/stdin/stderr.